src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -231,14 +238,34 @@ public void showOutputSummary() {
         isFailure ? "\u001B[1;31m" : "\u001B[1;32m",
         isFailure ? "\u001B[1;35m" : "\u001B[1;32m", log);
 
-    uploadOutputSummary(failed, completedChecks);
+    // Comment out here, for this step is used directly in the report Task
+    // uploadOutputSummary(failed, completedChecks);
 
     if (isFailure && GatorGradleConfig.get().shouldBreakBuild()) {
       throw new GradleException(
           StringUtil.color(StringUtil.BAD, "Grading checks failed -- scroll up for failures"));
     }
   }
 
+  /**
+   * Return the completed checks.
+   */
+  public List<CheckResult> getCompletedchecks() {
```

Comment:
`completedChecks` is an internal variable, and should not be exposed. Instead, `uploadSummaryReport` should probably not take the checks to upload as a parameter, but instead use the internal class variables. An even better solution would be to make two versions of `uploadSummaryReport`, one of which takes the parameters as now, and the other taking nothing and using the internal member variables to call the previous one. That way we preserve the ability to upload only parts of a summary if we wish in the future, while not exposing encapsulated variables.